velox/expression/type_calculation/TypeCalculation.ll

```diff
@@ -22,6 +22,14 @@ var             ([[:alpha:]][[:alnum:]_]*)
 "="             return Parser::token::ASSIGN;
 "min"           return Parser::token::MIN;
 "max"           return Parser::token::MAX;
+"<"             return Parser::token::LT;
```

Comment:
This will enhance the function signature, so @majetideepak suggests me to add it in this PR, https://github.com/facebookincubator/velox/issues/4773#issuecomment-1536073339

Do you make sure we should extract these into a separate PR?

Reply:
Hmm... Can you describe these change in the PR description? Are there tests for these? Can you give me a pointer?

Reply:
https://github.com/facebookincubator/velox/pull/4613/files#diff-4baee9f82f9c347f753972415d345941d2c2a110b608d480b090650171da096bR379
It is used to compute the result scale in function `getResultScale`
```
std::string getResultScale(std::string precision, std::string scale) {
  return fmt::format(
      "({}) <= 38 ? ({}) : max(({}) - ({}) + 38, min(({}), 6))",
      precision,
      scale,
      scale,
      precision,
      scale);
}
```
Current Decimal multiply and divide test can cover `<=` and `?` changes, I also extend the calculation to support basic operator like `<`. 

If the computation is wrong, we cannot get the correct result type.

Reply:
Got it. Let's extract these changes into a separate PR and add a test to velox/expression/tests/SignatureBinderTest.cpp to cover all of added functionality.